lms/templates/courseware/course_updates.html

```diff
@@ -1,48 +1,18 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
-<div class="recent-updates">
-  % for index, update in enumerate(visible_updates):
+<div class="all-updates">
+  % for index, update in enumerate(updates):
     <article class="updates-article">
       % if not update.get("is_error"):
-        <h5 class="date" id="msg-date-${index}">${update.get("date")}</h5>
-        <button
-            class="toggle-visibility-button"
-            data-hide="${_('Hide')}"
-            data-show="${_('Show')}"
-            aria-describedby="msg-date-${index}"
-            aria-controls="msg-content-${index}"
-            aria-expanded="true"
-        ></button>
+        <div class="date" id="msg-date-${index}">${update.get("date")}</div>
+      % endif
+        <div class="article-content" id="msg-content-${index}">
+          ${update.get("content")}
+        </div>
+      % if (index < len(updates) - 1):
```

Comment:
I wouldn't add a div just for styling. If you want to add a line everywhere but the last one, you can use something like:

    border-bottom: 1px solid $lms-border-color;

    &:last-child {
        border-bottom: 0;
    }